release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1064,6 +1081,35 @@ You can attempt to resolve the issue with the steps in this link:https://kb.vmwa
 
 * An Open Virtual Network (OVN) bug causes persistent connectivity issues with Octavia load balancers. When Octavia load balancers are created, OVN might not plug them into some Neutron subnets. These load balancers might be unreachable for some of the Neutron subnets. This problem affects Neutron subnets, which are created for each OpenShift namespace, at random when Kuryr is configured. As a result, when this problem occurs the load balancer that implements OpenShift `Service` objects will be unreachable from OpenShift namespaces affected by the issue. Because of this bug, {product-title} 4.8 deployments that use Kuryr SDN are not recommended on {rh-openstack-first} 16.1 with OVN and OVN Octavia configured until the bug is fixed. To track this issue, see (link:https://bugzilla.redhat.com/show_bug.cgi?id=1937392[*BZ#1937392*]).
 
+* If you are using {sandboxed-containers-first}, you cannot use the `hostPath` volume in a {product-title} cluster to mount a file or directory from the host node’s file system into your pod.
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1904609[*BZ#1904609*])
+
+* If you are running Fedora on {sandboxed-containers-first}, you need a workaround to install some packages. Some packages, like `iputils`, require file access permission changes that {product-title} does not grant to containers by default. To run containers that require such special permissions, it is necessary to add an annotation to the YAML file describing the workload, which tells `virtiofsd` to accept such file permissions for that workload. The required annotations are:
++
+[source,yaml]
+----
+io.katacontainers.config.hypervisor.virtio_fs_extra_args: [ "-o", "modcaps=+sys_admin", "-o", "xattr" ]
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1915377[*BZ#1915377*])
+
+* In the 4.8 release, the {sandboxed-containers-operator} does not handle the custom `MachineConfigPool`. Attempting to use customized `MachineConfigPool` may result in consistent behavior while installing and uninstalling the Operator.
```

Comment:
The problem here is that the actual feature (selecting nodes for the installation by adding a label to it) does work, but the UI that we present to the user is wrong. It can be worked around by adding the kataConfigPoolSelector manually. I suggest to add:

The impact of this is that the scheduling.nodeSelector will not be set, which means Pods using the
runtime class 'kata' could be scheduled to Nodes that don’t have the Kata Containers runtime installed.
This problem can be worked around by specifying the kataConfigPoolSelector manually in the kataconfig. In a terminal do 'oc edit kataconfig example-kataconfig' then add the selector.  A full example of a kataconfig CR

```
apiVersion: kataconfiguration.openshift.io/v1
kind: KataConfig
metadata:
  name: example-kataconfig
  spec:
    kataConfigPoolSelector:
      matchLabels:
        custom-kata1: test
```